test/functional/plugin/lsp_spec.lua

```diff
@@ -861,4 +861,29 @@ describe('LSP', function()
       ]])
     end)
   end)
+  describe('lsp.util.show_line_diagnostics', function()
+    it('show_line_diagnotics shows diagnostics for current line', function()
```

Comment:
can omit `show_line_diagnotics` (also typo) in the it(), since it() descriptions are combined with their describe() descriptions.
```suggestion
    it('shows diagnostics for current line', function()
```